lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -966,6 +967,23 @@ listByronTransactions
 listByronTransactions =
     listTransactions
 
+deleteByronTransaction
+    :: forall ctx s t k.
+        ( s ~ RndState t
+        , ctx ~ ApiLayer s t k
+        , DefineTx t
+        )
+    => ctx
+    -> ApiT WalletId
+    -> ApiTxId
+    -> Handler NoContent
+deleteByronTransaction ctx (ApiT wid) (ApiTxId (ApiT (tid))) = do
+    liftHandler $ withWorkerCtx ctx wid liftE $ \wrk ->
+        W.forgetPendingTx wrk wid tid
+    return NoContent
+  where
+    liftE (ErrNoSuchWallet wid') = throwE $ ErrRemovePendingTxNoSuchWallet wid'
```

Comment:
`ErrRemovePendingTxNoSuchWallet` should take an `ErrNoSuchWallet` as argument instead of `WalletId` to avoid this awkward unwrapping/wrapping.

Reply:
ok, corrected